sagemaker-training-compiler/huggingface/pytorch_tune_batch_size/batch_tune.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>This notebook uses a HuggingFace training script to demonstrate how to find the maximum batch size that can optimally fit into GPU memory. If you want to test with your own training script, you need to update the following:</p><p> </p><ul><li>The<code>find_max_batch_size.py</code> script - In line 23 to 28 of the script, specify the right directory path and the file name of your training script.</li><li><code>hyperparameters</code> - In the following <strong>Tune a Native PyTorch Training Job</strong>, modify the hyperparameters that your training script requires accordingly.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Worth clarifying somewhere within the notebook intro/setup as to why it's beneficial to find the max batch size that can fit into memory - I assume to minimize the training time.</p>